fsleyes-plugin-shimming-toolbox/fsleyes_plugin_shimming_toolbox/tabs/mask_tab.py

```diff
@@ -221,11 +232,129 @@ def create_sizer_sphere(self):
                 "name": "output",
             }
         ]
-        component = InputComponent(self, input_text_box_metadata, sphere)
+        component = InputComponent(self, input_text_box_metadata, cli=sphere)
         self.run_component_sphere = RunComponent(
             panel=self,
             list_components=[component],
             st_function="st_mask sphere"
         )
         sizer = self.run_component_sphere.sizer
         return sizer
+    
+    def create_sizer_bet(self):
+        path_output = os.path.join(__CURR_DIR__, "output_mask_bet")
+        input_text_box_metadata = [
+            {
+                "button_label": "Input",
+                "button_function": "select_from_overlay",
+                "name": "input",
+                "required": True
+            },
+            {
+                "button_label": "fractional intensity threshold",
+                "name": "f_param",
+                "default_text": "0.5",
+                "required": False
+            },
+            {
+                "button_label": "Vertical gradient",
+                "name": "g_param",
+                "default_text": "0.0",
+                "required": True
+            },
+            {
+                "button_label": "Output File",
+                "button_function": "select_folder",
+                "default_text": os.path.join(path_output, "bet.nii.gz"),
+                "name": "output",
+            }
+        ]
+        component = InputComponent(self, input_text_box_metadata, cli=bet)
+        self.run_component_bet = RunComponent(
+            panel=self,
+            list_components=[component],
+            st_function="st_mask bet"
+        )
+        sizer = self.run_component_bet.sizer
+        return sizer
+    
+    def create_sizer_modify(self):
+        path_output = os.path.join(__CURR_DIR__, "output_mask_modify")
+        input_text_box_metadata_1 = [
+            {
+                "button_label": "Input",
+                "button_function": "select_from_overlay",
+                "name": "input",
+                "required": True
+            }
+        ]
+        dropdown_operation_metadata = [
+            {
+                "label": "Erode",
+                "option_value": "erode"
+            },
+            {
+                "label": "Dilate",
+                "option_value": "dilate"
+            }
+        ]
+        dropdown_shape_metadata = [
+            {
+                "label": "Sphere",
+                "option_value": "sphere"
+            },
+            {
+                "label": "Cross",
+                "option_value": "cross"
+            },
+            {
+                "label": "Line",
+                "option_value": "line"
+            },
+            {
+                "label": "Cube",
+                "option_value": "cube"
+            },
+            {
+                "label": "None",
+                "option_value": "None"
+            }
```

Comment:
None should probably not be an option in the GUI as it does nothing
```suggestion
            }
```